File: liquidacion.py

SALARIO_BASE = 2000000
DIAS_LABORADOS = 300


prima = (SALARIO_BASE * DIAS_LABORADOS) / 360
cesantias = (SALARIO_BASE * DIAS_LABORADOS) / 360
vacaciones = (SALARIO_BASE * DIAS_LABORADOS) / 720
intereses_cesantias = ((cesantias * 0.12) * (DIAS_LABORADOS)) / 360
total = prima+cesantias+vacaciones+intereses_cesantias
print(round(total,0))

File: Liquidacion.java
public class Liquidacion {

    public static void main(String[] args) {

        float salarioBase = 2000000;
        float diasLaborados = 300;
        float prima = (salarioBase * diasLaborados) /360;
        float cesantias = (salarioBase * diasLaborados) / 360;
        float vacaciones = (salarioBase * diasLaborados) / 720;
        float interesesCesantias = (float) (((cesantias * 0.12)*diasLaborados)/360);
        float total = prima + cesantias+vacaciones+interesesCesantias;
        System.out.println("El salario total es: " + Math.round(total));
    }

}
